package.json

```diff
@@ -97,6 +97,7 @@
     "compression": "^1.7.4",
     "connect-flash": "^0.1.1",
     "connect-redis": "^6.0.0",
+    "content-disposition": "^0.5.4",
```

Comment:
@AndyNewton just checking - so this library is doing two things for us - defaulting to 'attachment', and encoding the filename?

Reply:
Yes.  It is this: https://github.com/jshttp/content-disposition#readme
In fact as we can see from the tests it results in a call to `res.header()` i.e. from:
`expect(res.header).toHaveBeenCalledWith( ... )`